packages/clis/generator/geo-json.ts

```diff
@@ -833,6 +855,339 @@ export function convertToContoursGeoJson({
   } as const;
 }
 
+function calcDlcGuard(startDlc: number, endDlc: number) {
```

Comment:
This will likely need to change once ETS2 achievements are added (assuming ETS2 has achievements that require taking a ferry between two different countries).